docker-compose.yml

```diff
@@ -283,6 +289,19 @@ services:
       timeout: 5s
       retries: 5
 
+  kafka-ui:
```

Comment:
Can't wait to see what it looks like, you are the master of dev tools :P Kudos to you!